### PR TITLE
Feature/improve notebook admin layout

### PR DESCRIPTION
**Pull request by takvamborgen**
- Move note action buttons (delete, open) to mouseover on note element in tree
- Remove JSON metadata
- Add display of each paragraph (UI needs improvement)

### CODE REVIEW

File: src/pages/prep_and_analysis/NotebookAdmin.js [hunk 1]
```diff
@@ -201,21 +204,50 @@ class NotebookAdmin extends Component {
...
+    }, () => {
+      const { user } = this.props
+      const { noteToDelete } = this.state
+
+      let context = this.context
+
+      context.notebookService.deleteNote(noteToDelete.id, user).then(() => {
+        this.setState({
+          deleted: true,
+          message: 'Was deleted'
```
Comment (bjornandre) [inline src/pages/prep_and_analysis/NotebookAdmin.js]: Move text to properties file

File: src/pages/prep_and_analysis/NotebookAdmin.js [hunk 2]
```diff
@@ -201,21 +204,50 @@ class NotebookAdmin extends Component {
...
+        })
+        this.loadNotes() // TODO remember folder collapsed status
+      }).catch(error => {
+        this.setState({
+          deleted: false,
+          error: true,
+          message: `Could not delete (${error})`
```
Comment (bjornandre) [inline src/pages/prep_and_analysis/NotebookAdmin.js]: Move text to properties file

File: src/pages/prep_and_analysis/NotebookAdmin.js (hunk 1 above)
Comment (mmj-ssb) [reply src/pages/prep_and_analysis/NotebookAdmin.js]: Agree

File: src/pages/prep_and_analysis/NotebookAdmin.js (hunk 2 above)
Comment (mmj-ssb) [reply src/pages/prep_and_analysis/NotebookAdmin.js]: Agree

File: src/pages/prep_and_analysis/NotebookAdmin.js (hunk 1 above)
Comment (takvamborgen) [reply src/pages/prep_and_analysis/NotebookAdmin.js]: Fixed

File: src/pages/prep_and_analysis/NotebookAdmin.js (hunk 2 above)
Comment (takvamborgen) [reply src/pages/prep_and_analysis/NotebookAdmin.js]: Fixed

**Pull request merged**